apps/web/src/pages/Applications/ApplicantDashboardPage/components/TrackApplications/TrackApplicationsStatus.tsx

```diff
@@ -154,57 +160,62 @@ const QualifiedRecruitmentStatus = ({
         {intl.formatMessage({
           defaultMessage:
             "You have chosen not to appear in talent results for this recruitment.",
-          id: "bqWmTD",
+          id: "KrAvuk",
           description:
-            "Suspended recruitment status message on qualified recruitment card.",
+            "Suspended recruitment status message on application card.",
         })}
-        {/* Dialog that allows user to un-suspend the qualified recruitment (application). Sets suspendedAt date to null. */}
-        <QualifiedRecruitmentDialog
+        {/* Dialog that allows user to un-suspend the application. Sets suspendedAt date to null. */}
+        <TrackApplicationsDialog
           id={id}
           isSuspended={!!suspendedAt}
           title={intl.formatMessage({
             defaultMessage: "Change your appearance in search results",
-            id: "FGF3sd",
-            description:
-              "Dialog title on change search results status from qualified recruitment section.",
-          })}
-          primaryBodyText={intl.formatMessage({
-            defaultMessage:
-              "You are currently not showing up in talent search results for the “IT-01 Helpdesk support technician” pool.",
-            id: "veIkpN",
+            id: "3qdrpr",
             description:
-              "Dialog main body on change search results status from qualified recruitment section.",
+              "Dialog title on change search results status from applications section.",
           })}
+          primaryBodyText={intl.formatMessage(
+            {
+              defaultMessage:
+                "You are currently not showing up in talent search results for the “{poolName}” pool.",
```

Comment:
Another use of smart quotes we could get rid of.

Reply:
Right, I can easily fix that now.
Ref: [fix smart quotes](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6823/commits/42020a4701ca9330737cdb03b484ec1e4513d3c7)